guides/common/docinfo.html

```diff
@@ -1 +1,74 @@
-<!-- docinfo -->
+<!-- Generate a nice TOC -->
+<script src="/js/jquery-1.11.3.min.js"></script>
+<script src="/js/jquery-ui.min.js"></script>
+<script src="/js/jquery.tocify.min.js"></script>
```

Comment:
This JS path isn't working and I don't know what the correct path is.

Reply:
@ekohl Would you be able to help?

Reply:
@maximiliankolb If you know who to ask for help, can you, please? :)

Reply:
@Lennonka I tested your change locally via `make serve` and found it to work nicely!
Checking the [pipeline log](https://github.com/theforeman/foreman-documentation/actions/runs/11843279429/job/33003853408?pr=3453#step:2:68), I noticed that the deployment was using commit `398142fa8e6130035e761c59b8e0632867e38455`. This obviously does not have js libraries, which results in errors.
I don't know why this happens, but it should not block this PR.